nixos/modules/services/security/clamav-unofficial-sigs.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.clamav-unofficial-sigs;
+in {
+  options.services.clamav-unofficial-sigs = with types; {
+    enable = mkEnableOption "Unofficial ClamAV signatures";
+
+    user = mkOption {
+      type = str;
+      description = "User to run systemd service under";
+      default = "clamav";
+    };
+
+    group = mkOption {
+      type = str;
+      description = "Group to run systemd service under. Should be the same as used by clamav in order to allow this service access to the clamav database.";
+      default = "clamav";
+    };
+
+    extraConfig = mkOption {
+      type = attrsOf str;
+      description = "Extra key-value pairs to configure in user.conf";
+      example = { malwarepatrol_receipt_code = "YOUR-RECEIPT-NUMBER"; };
+      default = {};
+    };
+
+    extraConfigFiles = mkOption {
+      type = listOf str;
+      description = "Extra configuration files to include";
+      default = [];
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.tmpfiles.rules = [
+      "d /var/lib/secrets/clamav-unofficial-sigs 0700 ${cfg.user} ${cfg.group} -"
```

Comment:
Where is this directory used? I don't see any references.

Reply:
That was a mistake, I removed it